### PR TITLE
Update code base

### PR DESCRIPTION
**Pull request by NilshanDeemantha**

### CODE REVIEW

File: web/src/App.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+import "./App.css";
 
-function App() {
+const App = () => {
   return (
     <div className="App">
       <BrowserRouter>
         <Routes>
-          <Route path="*" element={<Login />} />
           <Route path="/login" element={<Login />} />
```
Comment (buddhi-system2) [inline web/src/App.js:12]: what happens if we goto localhost:3000 now?

Comment (NilshanDeemantha) [reply web/src/App.js:12]: Display an empty window in browser

Comment (buddhi-system2) [reply web/src/App.js:12]: It should be redirected to login page, can you please do that change

Comment (NilshanDeemantha) [reply web/src/App.js:12]: Changes are done and push to the branch

Comment (buddhi-system2) [reply web/src/App.js:12]: we need to redirect to /login not just show login component

Comment (NilshanDeemantha) [reply web/src/App.js:12]: Changes are done . If we request http://localhost:3000/ now, it navigate to login page and changing the URL to http://localhost:3000/login

**Review by buddhi-system2 — APPROVED** [review]

**Pull request merged**